inst/tests/sqlite.Rraw

```diff
@@ -0,0 +1,306 @@
+# mergelist join tester vs SQLite, based on v1.9.8 non-equi join tester
+
+if (!all(
+  packageVersion("RSQLite") > "1.0.0", ## we use window function which is not available in RSQLite v1.0.0 (sqlite 3.8.6)
+  packageVersion("RSQLite") < "1.1.0"  ## we don't want upstream RSQLite because it is not lite anymore
+))
+  stop('Use RSQLite which is up-to-date and lightweight:\ninstall.packages(c("DBI","RSQLite"), repos="https://jangorecki.gitlab.io/rsqlite")')
+
+# as of now run by: Rscript inst/tests/sqlite.R
+#library(data.table); shallow = data.table:::shallow
+cc(quiet=TRUE)
```

Comment:
is this left here on purpose?

Reply:
This file is not being run unless user calls `Rscript inst/tests/sqlite.R`. It depends on my fork of RSQLite. I could eventually add it to one of the CI jobs, the one that runs `others.Rraw`.